posthog/temporal/data_imports/deltalake_compaction_job.py

```diff
@@ -39,7 +40,11 @@ def trigger_compaction_job(job: ExternalDataJob, schema: ExternalDataSchema) ->
 
         if not DEBUG and not TEST:
             # Wait for the compaction to complete before continuing
-            asyncio.run(handle.result())
+            try:
+                asyncio.run(handle.result())
+            except Exception as e:
+                capture_exception(e)
+                logger.exception(f"Compaction job failed with: {e}", exc_info=e)
```

Comment:
Don't we want to re-raise here? Otherwise this will return a workflow-id of a Workflow that failed. But it looks like we are only logging that ID so maybe it's better not to re-raise.

Reply:
Nah, we want to carry on the external data job as normal - we want to avoid compaction job errors making the external data job fail

Reply:
Makes sense, thank you!